resources/views/layouts/partials/header/account.blade.php

```diff
@@ -1,19 +1,22 @@
 <div class="mr-3">
-    <toggler v-if="loggedIn" v-cloak>
+    <toggler v-if="$root.loggedIn" v-cloak>
         <div slot-scope="{ toggle, close, isOpen }" v-on-click-away="close">
             <button dusk="account_menu" class="flex my-1" v-on:click="toggle">
                 <x-heroicon-o-user class="h-6 w-6"/>
                 @{{ $root.user.firstname }}
             </button>
             <div v-if="isOpen" class="absolute bg-white border shadow rounded mr-1 {{ config('rapidez.z-indexes.header-dropdowns') }} {{ Route::currentRouteName() == 'checkout' ? 'right-0' : '' }}">
                 @if(App::providerIsLoaded('Rapidez\Account\AccountServiceProvider'))
-                    <a class="block hover:bg-secondary px-3 py-2" href="{{ route('account.overview') }}">@lang('Account')</a>
-                    <a class="block hover:bg-secondary px-3 py-2" href="{{ route('account.orders') }}">@lang('Orders')</a>
+                    <a class="block hover:bg-inactive px-3 py-2" href="{{ route('account.overview') }}">@lang('Account')</a>
+                    <a class="block hover:bg-inactive px-3 py-2" href="{{ route('account.orders') }}">@lang('Orders')</a>
+                    @if (App::providerIsLoaded('Rapidez\Wishlist\WishlistServiceProvider'))
+                        <a class="block hover:bg-inactive px-3 py-2" href="{{ route('account.wishlist') }}">@lang('Wishlist')</a>
+                    @endif
                 @endif
                 <user>
                     <a
                         href="#"
```

Comment:
can this be a <button? since the href is not used? that the @click can be changed without the .prevent right?